src/CTA.Rules.Models/Actions/Attributeaction.cs

```diff
@@ -8,7 +8,7 @@ public class AttributeAction : GenericAction
     {
         public Func<SyntaxGenerator, AttributeSyntax, AttributeSyntax> AttributeActionFunc { get; set; }
         public Func<SyntaxGenerator, AttributeListSyntax, AttributeListSyntax> AttributeListActionFunc { get; set; }
-
+        public new AttributeAction Clone() => (AttributeAction)this.MemberwiseClone();
```

Comment:
Instead of having a Clone() in each action, can we add a generic clone in the base class? Something like:

public T Clone<T>() => (T)this.MemberwiseClone();

This way we can reuse it in all the child classes by passing the type we are cloning. This applies to all the *Action updates.

Reply:
Good point, will add the clone method to the base action class instead.